public/css/dashboard.css

```diff
@@ -93,20 +93,42 @@ a {
     /*display:none;*/
 }
 
+.filter .flatpickr-input[readonly] {
```

Comment:
I don't think this is related to flatpickr (the datepicker component) or the fact that it marks the input as read only.

The targeted selector should (please confirm) probably just be something like `.filter input[type=text]` or even `.filter input`.